CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## Current (in progress)
```

Comment:
Maybe a `0.0.1` here?

Reply:
I think `bumpr` will do this for me when we release, right @noirbizarre?